Apps/Database/src/Models/UserFeedbackAdmin.cs

```diff
@@ -0,0 +1,31 @@
+﻿//-------------------------------------------------------------------------
+// Copyright © 2019 Province of British Columbia
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//-------------------------------------------------------------------------
+namespace HealthGateway.Database.Models
+{
+    using System.ComponentModel.DataAnnotations;
+
+    /// <summary>
+    /// An administrative view of UserFeedback with Email populated from UserProfile.
+    /// </summary>
+    public class UserFeedbackAdmin : UserFeedback
```

Comment:
Will this not create a new table?

Reply:
No, to create a new table it would need to be added to the DB Context and even then it would create an alter to the UserFeedback table and add a discriminator as its a child object.